backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: MIT
+# SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import os
+import json
+
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+"""
+export APPWRITE_ENDPOINT=<http://localhost/v1>
+export APPWRITE_PROJECT=<project_id>
+export APPWRITE_API_KEY=<api_key>
+"""
+APPWRITE_ENDPOINT = os.environ.get("APPWRITE_ENDPOINT")
+APPWRITE_PROJECT = os.environ.get("APPWRITE_PROJECT")
+APPWRITE_API_KEY = os.environ.get("APPWRITE_API_KEY")
+
+
+client = Client()
+database = Database(client)
+
+(
+    client.set_endpoint(APPWRITE_ENDPOINT)
+    .set_project(APPWRITE_PROJECT)
+    .set_key(APPWRITE_API_KEY)
+)
+
+
+createCollectionResult = database.create_collection(
+    "Announcements",  # Collection Name
+    ["*"],  # Read permissions
+    ["team:Admins"],  # Write permissions
+    [
+        {
+            "label": "created_at",
+            "key": "created_at",
+            "type": "numeric",
+            "required": True,
+            "array": False,
+        },
+        {
+            "label": "updated_at",
+            "key": "updated_at",
+            "type": "numeric",
+            "required": False,
+            "array": False,
+        },
+        {
+            "label": "title",
+            "key": "title",
+            "type": "text",
+            "required": True,
+            "array": False,
+        },
+        {
+            "label": "content",
+            "key": "content",
+            "type": "text",
+            "required": True,
+            "array": False,
+        },
+    ],
+)
```

Comment:
Would you mind if I commit this suggestion?
```suggestion
collection = [
	{
	    "label": "created_at",
	    "key": "created_at",
	    "type": "numeric",
	    "required": True,
	    "array": False,
	},
	{
	    "label": "updated_at",
	    "key": "updated_at",
	    "type": "numeric",
	    "required": False,
	    "array": False,
	},
	{
	    "label": "title",
	    "key": "title",
	    "type": "text",
	    "required": True,
	    "array": False,
	},
	{
	    "label": "content",
	    "key": "content",
	    "type": "text",
	    "required": True,
	    "array": False,
	},
]

createCollectionResult = database.create_collection(
    collection_name := "Announcements",
    read_permissions := ["*"],
    write_permissions := ["team:Admins"],
    collection,
)
```